src/sentry/auth/access.py

```diff
@@ -175,111 +182,63 @@ def has_project_scope(self, project, scope):
         """
         return self.has_project_access(project) and self.has_scope(scope)
 
-    def to_django_context(self):
+    def to_django_context(self) -> Mapping[str, bool]:
         return {s.replace(":", "_"): self.has_scope(s) for s in settings.SENTRY_SCOPES}
 
 
-class Access(BaseAccess):
```

Comment:
@RyanSkonnord So no more `Access` class?

Reply:
No, the declaration is on line 80 now.